app/models/user.rb

```diff
@@ -86,6 +86,24 @@ def mailboxer_email(object)
     self.email
   end
 
+  def self.import file 
```

Comment:
use class << sefl to define class method

Reply:
Done